spec/graphql/schema/argument_spec.rb

```diff
@@ -167,47 +169,60 @@ def self.object_from_id(id, ctx)
   describe "default_value:" do
     it 'uses default_value: with no input' do
       query_str = <<-GRAPHQL
-      { field }
+      { field(requiredArg: 1) }
       GRAPHQL
 
       res = SchemaArgumentTest::Schema.execute(query_str)
-      assert_equal '{:required_with_default_arg=>1}', res["data"]["field"]
+      assert_equal '{:default_value_arg=>1, :required_arg=>1}', res["data"]["field"]
     end
 
     it 'uses provided input value' do
       query_str = <<-GRAPHQL
-      { field(requiredWithDefaultArg: 2) }
+      { field(defaultValueArg: 2, requiredArg: 1) }
       GRAPHQL
 
       res = SchemaArgumentTest::Schema.execute(query_str)
-      assert_equal '{:required_with_default_arg=>2}', res["data"]["field"]
+      assert_equal '{:default_value_arg=>2, :required_arg=>1}', res["data"]["field"]
     end
+  end
 
+  describe 'required:' do
     it 'respects non-null type' do
       query_str = <<-GRAPHQL
-      { field(requiredWithDefaultArg: null) }
+      { field(requiredArg: null) }
       GRAPHQL
 
       res = SchemaArgumentTest::Schema.execute(query_str)
-      assert_equal "Argument 'requiredWithDefaultArg' on Field 'field' has an invalid value (null). Expected type 'Int!'.", res['errors'][0]['message']
+      assert_equal "Argument 'requiredArg' on Field 'field' has an invalid value (null). Expected type 'Int!'.", res['errors'][0]['message']
+    end
+  end
+
+  describe 'invalid:' do
+    it 'field has conflicting params' do
```

Comment:
how about changing this description to work with `it` in a sentence, to make things more readable:
```suggestion
    it 'raises an error when a field has conflicting params' do
```